docs/Training-ML-Agents.md

```diff
@@ -78,7 +78,7 @@ training run if you plan to view the statistics. You can view these statistics
 using TensorBoard during or after training by running the following command:
 
 ```sh
-tensorboard --logdir=summaries
+tensorboard --logdir=summaries --port 6006
```

Comment:
We need to add a comment explaining the port number.

Reply:
Or at least say that the port parameter is explained on the Using-TensorBoard page.

Reply:
Added the same comment as on the Using-TensorBoard page.

Reply:
I'd rather keep this part simple and have the more detailed usage in Using-TensorBoard. I don't think the note on the port belongs here.